com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/persistence/query/ConfigurationSubscriptionQuery.java

```diff
@@ -19,4 +20,12 @@
 
     ConfigurationSubscriptionQuery onSelectMatching(List<ConfigurationEntry> entries);
 
+    ConfigurationSubscriptionQuery moduleIdNull();
```

Comment:
All new methods except for `deleteAll` aren't used anywhere (at least as far as I can see). Can we remove them from the PR?